templates/partials/filters/report-type.html

```diff
@@ -1,211 +1,211 @@
 <div class="filter">
```

Comment:
Is there a reason this isn't being done in a loop? Seems like hteres a lot of repetition in this file.